### PR TITLE
DS-2 QoL / Interdyne light fix

### PR DESCRIPTION
**Pull request by LovliestPlant**
## About The Pull Request

This PR scoots the tube light in Interdyne's (both lavaland and icebox) cafeteria up by a tile. It is no longer floating above the kitchen counter. Additionally it replaces the sheet of plastitanium and plastitanium glass in DS-2's engine room with a stack of 20 plasteel sheets and 20 titanium sheets for use in upgrading DS-2's turbine turbine. Lastly it adds a packet of korta seeds to DS-2's permabrig, as a treat.

## How This Contributes To The Skyrat Roleplay Experience

The floating light looked a bit wonky, sits nicer on the wall.

DS-2 is currently unable to power itself, even under idle load, with it's roundstart turbine. Adding some plasteel and titanium will allow players to upgrade the area's turbine to T2, which is sufficient to meet base power needs.

Food options for lizards, moths, and other diet restricted species are pretty scarce in DS-2's permabrig. Average population being what it is there, it's not a given to have a chef to make food for prisoners. Adding some korta nuts to the mix will open up a wide range of possibilities for players who sign on as DS-2 hostages, and should make life a bit less frustrating.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
Interdyne Lights
Before
ID_before

After
ID_Light

DS-2 Engine Room
DS_2-Plasteel

DS-2 Permabrig
DS_2-Korta

</details>

## Changelog

:cl:
qol: [DS-2] Replaces the pair of plastitatinum sheets with small stacks of plasteel and titanium in the engine room.
qol: [DS-2] Adds a korta seed packet to the botany box in the permabrig.
fix: [Interdyne] Fixes the floating tube light above the kitchen counter in the cafeteria.
/:cl:

### CODE REVIEW

File: _maps/RandomRuins/SpaceRuins/skyrat/interdynefob.dmm [hunk 1]
```diff
@@ -9805,7 +9805,7 @@
 "Tm" = (
 /obj/machinery/atmospherics/pipe/smart/manifold4w/scrubbers/hidden/layer2,
 /obj/machinery/computer/atmos_control/nocontrol/incinerator{
-	atmos_chambers = list("syndieincinerator"="DS-2 Incinerator Chamber");
+	atmos_chambers = list("syndieincinerator"="DS-2  Incinerator  Chamber");
```
Comment (GoldenAlpharex) [inline _maps/RandomRuins/SpaceRuins/skyrat/interdynefob.dmm]: I don't think this change is needed.
```suggestion
	atmos_chambers = list("syndieincinerator"="DS-2 Incinerator Chamber");
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Outside of my one requested change, good changes!

**Comment by LovliestPlant** [comment]
After a bit more tinkering I've found that the turbine outlet also needs to be upgraded to reliably meet power needs. Added a stack of 20 titanium next to the plasteel for that task.

**Comment by vinylspiders** [comment]
@GoldenAlpharex  good to go?

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**